web/src/click_handlers.ts

```diff
@@ -265,6 +270,21 @@ export function initialize(): void {
         e.preventDefault();
     });
 
+    $("#main_div").on("click", ".rehide-muted-user-message", (e) => {
+        const message_id = Number($(e.currentTarget).attr("data-message-id"));
+        assert(message_lists.current !== undefined);
+        const $row = message_lists.current.get_row(message_id);
+        const message = message_lists.current.get(rows.id($row));
+        assert(message !== undefined);
+        const message_container = message_lists.current.view.message_containers.get(message.id);
+        assert(message_container !== undefined);
+        if ($row && !message_container.is_hidden) {
+            message_lists.current.view.hide_revealed_message(message_id);
```

Comment:
When would this conditional fail? I would have thought this button was only visible when this should pass; should it be an assertion?